standards/11-clipboard.md

```diff
@@ -0,0 +1,41 @@
+# *COS 11:* Clipboard
+
+## Quick information
+| Information |                                                                                        |
+| ----------- | -------------------------------------------------------------------------------------- |
+| Version     | 1.0.0                                                                                  |
+| Type        | Protocol                                                                               |
+
+### Technical Details
+Clipboard defines a method to share information like text with other programs.
+
+It is a Table called "clipboard" in the Shell Environment.
```

Comment:
> It is a table called "clipboard" in the shell environment.

Nouns shouldn't be capitalised. There's plenty other instances of this in the document, which I won't comment on to avoid clutter.